### DISCUSSION TITLE
New Online Accounts

### DISCUSSION THREAD

**Discussion by DSLitvinov** [elementary/switchboard-plug-onlineaccounts#267] [Ideas]
Hi, how about simplifying the connection of online accounts? And everything is made much easier for a Gnome, and this point greatly hinders calling an elementary OS elementary. Maybe you should include gnome-online-accounts with your frontend?

**Comment by lenemter** [comment]
That's a great idea and I got a working prototype here: https://github.com/elementary/switchboard-plug-onlineaccounts/pull/268
